### PR TITLE
docs: misc changes while translating (substuting #2151 - smoe:docs29_mac_6_was_equiv_against_master)

### PR DESCRIPTION
**Pull request by smoe**
Reapplied https://github.com/LinuxCNC/linuxcnc/pull/2151 to branch 2.9.

### CODE REVIEW

**Comment by smoe** [comment]
@hansu - conflicts resolve.
Every accepted patch brings new conflicts, but, well, it is not too bad.

**Comment by smoe** [comment]
The conflicts look very much like what I already did for docs29_mac_2 - seems like I had not properly branched of master when I created this branch. I supposed to wait for docs29_mac_2 to arrive in the archive and this one may then be obsolete.

**Comment by smoe** [comment]
Yup, in part I could indeed just leave the "current change".

**Comment by smoe** [comment]
@hansu - resolved conflict.

**Comment by smoe** [comment]
Resolved conflicts.

**Comment by smoe** [comment]
Just a rebase to trigger a rebuild.

**Comment by hansu** [comment]
conflicts...

**Comment by smoe** [comment]
@hansu - Removed conflict. In docs/src/plasma/qtplasmac.adoc I have also beauified the table with monospace a bit more.
Those changes are easy to do here, but mighty annoying for the translations, which is why I am doing them now and have the other languages spared from this.

**Comment by hansu** [comment]
Now you can continue resolving the conflicts here :D

**Comment by smoe** [comment]
@hansu - rebased and conflicts resolved.

File: docs/src/man/man1/xhc-whb04b-6.1.adoc [hunk 1]
```diff
@@ -306,44 +303,50 @@ MDI_COMMAND=(debug,macro16)
...
- see `whb.halui.spindle.`{`start`, `stop`}  True on S-ON/OFF button down, false otherwise.
+ see ``whb.halui.spindle.``{``start``, ``stop``}  True on S-ON/OFF button down, false otherwise.
  For toggling spindle on-off use `halui.spindle.0.start`.
  For toggling spindle on-off use `halui.spindle.0.stop`.
 `whb.button.fn` (bit,out)::
  True on Fn button down, false otherwise.
 `whb.button.probe-z` (bit,out)::
  connect to `halui.mdi-command-`__<M>__ True on Probe-Z button down, false otherwise.
- Requests MDI mode before button pin is set. See also *whb.halui.mode.mdi*.
+ Requests MDI mode before button pin is set. See also `*`whb.halui.mode.mdi`.
```
Comment (hansu) [inline docs/src/man/man1/xhc-whb04b-6.1.adoc:349]:
```suggestion
 Requests MDI mode before button pin is set. See also `whb.halui.mode.mdi`.
```

File: docs/src/plasma/plasma-cnc-primer.adoc [hunk 2]
```diff
@@ -519,11 +519,11 @@ Because the full scale is 10 Volts, then the frequency per Volt is:
 
 So assuming we have a 5 Volt input, the calculated frequency would be:
 
-(2520 Hz/V * 5 V) + 3,800 Hz = 16,400 Hz
+(2,520 Hz/V * 5 V) + 3,800 Hz = 16,400 Hz
```
Comment (hansu) [inline docs/src/plasma/plasma-cnc-primer.adoc:522]:
```suggestion
(2520 Hz/V * 5 V) + 3800 Hz = 16,400 Hz
```
IMHO these thousand separators are more confusing than helpful for such small numbers...

File: docs/man/man1/linuxcncrsh.1 [hunk 3]
```diff
@@ -351,7 +349,7 @@ With set, loads the tool table specified by <file>.
 .P
 \fBhome {0|1|2|...} | -1\fR
 .RS
-With set, homes the indicated joint or if -1, Home All joints.
+With set, homes the indicated joint or, if -1, homes All joints.
```
Comment (hansu) [inline docs/man/man1/linuxcncrsh.1:352]:
```suggestion
With set, homes the indicated joint or, if -1, homes all joints.
```

File: docs/src/plasma/qtplasmac.adoc [hunk 4]
```diff
@@ -2873,11 +2882,11 @@ These errors indicate the move to pierce height would exceed the Z axis maximum
 * *pierce height would exceed Z axis maximum safe height condition found while float switch probing*
 * *pierce height would exceed Z axis maximum safe height condition found while ohmic probing*
 
-=== Warning Errors
+=== Warnings
```
Comment (hansu) [inline docs/src/plasma/qtplasmac.adoc:2885]: Could also be "Warning messages"
```suggestion
=== Warning Messages
```

File: docs/src/plasma/qtplasmac.adoc [hunk 5]
```diff
@@ -3985,62 +4001,62 @@ There are some sample G-code files in the ~/linuxcnc/nc_files/examples/plasmac d
...
-M64 P2 (immediate)
-|Enable <<qt_thc,THC>> |M63 P2 (synchronized with motion) +
-M65 P2 (immediate)
-|Disable <<qt_overcut,Torch>> |M62 P3 (synchronized with motion) +
-M64 P3 (immediate)
-|Enable <<qt_overcut,Torch>> |M63 P3 (synchronized with motion) +
-M65 P3 (immediate)
-|Set <<qt_velocity_thc,velocity>> to a percentage of feed rate |M67 E3 Qn (synchronized with motion) +
-M68 E3 Qn (immediate) +
+|Set feed rate from <<qt_material-handling,material>> |`F#<_hal[plasmac.cut-feed-rate]>
```
Comment (hansu) [inline docs/src/plasma/qtplasmac.adoc:4016]: I guess here is a `` ` `` missing or similar

File: docs/src/plasma/qtplasmac.adoc [hunk 6]
```diff
@@ -3985,62 +4001,62 @@ There are some sample G-code files in the ~/linuxcnc/nc_files/examples/plasmac d
...
 *It is recommended to have M68 E3 Q0 in both the preamble and postamble.*
-|Cutter <<qt_cutter-compensation,compensation>> - left of path |G41.1 D#<_hal[plasmac.kerf-width]>
-|Cutter <<qt_cutter-compensation,compensation>> - right of path |G42.1 D#<_hal[plasmac.kerf-width]>
-|Cutter <<qt_cutter-compensation,compensation>> off |G40 +
+|Cutter <<qt_cutter-compensation,compensation>> - left of path m|G41.1 D#<_hal[plasmac.kerf-width]>
+|Cutter <<qt_cutter-compensation,compensation>> - right of path m|G42.1 D#<_hal[plasmac.kerf-width]>
+|Cutter <<qt_cutter-compensation,compensation>> off |`G40` +
 *Note that M62 through M68 are invalid while cutter compensation is on*
-|Cut <<qt_hole-cutting,holes>> at 60% feed rate |#<holes> = 1 +
+|Cut <<qt_hole-cutting,holes>> at 60% feed rate m|`#<holes> = 1` +
```
Comment (hansu) [inline docs/src/plasma/qtplasmac.adoc:4039]:
```suggestion
|Cut <<qt_hole-cutting,holes>> at 60% feed rate |`#<holes> = 1` +
```
I guess the "m" is not needed here?

File: docs/src/remap/remap.adoc [hunk 7]
```diff
@@ -766,14 +766,14 @@ We'll use the following correspondence in our example:
...
 |===
-|iocontrol.0 pin  |motion pin
-|tool-prepare     |digital-out-00
-|tool-prepared    |digital-in-00
-|tool-change      |digital-out-01
-|tool-changed     |digital-in-01
-|tool-prep-number |analog-out-00
-|tool-prep-pocket |analog-out-01
-|tool-number      |analog-out-02
+|`iocontrol.0` pin    |`motion` pin
```
Comment (hansu) [inline docs/src/remap/remap.adoc:769]: why not a leading "m" in this line?

Comment (smoe) [reply docs/src/remap/remap.adoc:769]: because of the word "pin" that is not part of the pin name

File: docs/src/plasma/qtplasmac.adoc (hunk 5 above)
Comment (smoe) [reply docs/src/plasma/qtplasmac.adoc]: Have introduced an "m" and removed the "`".

File: docs/src/plasma/plasma-cnc-primer.adoc [hunk 8]
```diff
@@ -519,11 +519,11 @@ Because the full scale is 10 Volts, then the frequency per Volt is:
 
 So assuming we have a 5 Volt input, the calculated frequency would be:
 
-(2520 Hz/V * 5 V) + 3,800 Hz = 16,400 Hz
+(2520 Hz/V * 5 V) + 3800 Hz = 16,400 Hz
 
 So now it should be fairly clear how to convert the frequency to its voltage equivalent:
 
-Volts = (frequency (Hz) - 3,800 Hz) / 2,520 Hz/V
+Volts = (frequency [Hz] - 3,800 Hz) / 2,520 Hz/V
```
Comment (hansu) [inline docs/src/plasma/plasma-cnc-primer.adoc:526]: This I meant with "next occurrence that is not displayed here"
We should have it consistent.

```suggestion
Volts = (frequency [Hz] - 3800 Hz) / 2520 Hz/V
```

File: docs/src/remap/remap.adoc (hunk 7 above)
Comment (hansu) [reply docs/src/remap/remap.adoc:769]: ok

**Comment by hansu** [comment]
please resolve the conflict...

**Comment by smoe** [comment]
@hansu - resolved.

**Comment by smoe** [comment]
Ooops, I had overwritten your https://github.com/LinuxCNC/linuxcnc/commit/b4840a71579685fc774fa360b2eef9a10b3fcc79 - fixed that in https://github.com/LinuxCNC/linuxcnc/pull/2159/commits/2cc3cffd907ab93d611d6c48184e85040ae626ff .

**Pull request merged**